core/src/test/scala/filodb.core/query/ColumnFilterMapSpec.scala

```diff
@@ -127,4 +127,97 @@ class ColumnFilterMapSpec extends AnyFunSpec with Matchers {
       cfMap.get(Map("b" -> "bbb123", "c" -> "ccc123")).get shouldEqual "4r"
     }
   }
+
```

Comment:
Can you add a test case ?
`regex("c", "c.*cc.*")), "4r"),`

Reply:
Added tests to both `Default` and `Fast` maps ✅